### ISSUE TITLE
How to support with money?

### ISSUE THREAD

**Issue by romixch** [JetpackDuba/Gitnuro#168]
This is not really a feature request. But I like projects like this. I know it is a lot of effort that people do without seeing any money. I know people don't do it for cash. But it helps to keep projects running. I know it is a real motivation. And it is not that hard to set up a patreon account or buy me a coffee or anything.

So the wanted "feature" would be "Provide us with some link to buy you a coffee" :-)

**Comment by JetpackDuba** [comment]
Thank you for your interest :) [It was requested](https://github.com/JetpackDuba/Gitnuro/issues/110) in the past but I didn't accept it because I didn't really need it but... I've changed my idea, as the situation has changed a bit and I'd also like to invest in hardware to better support all the platforms.

I've done some research about it and GitHub sponsors seems to be the most reasonable option for users and myself. I've enabled the [sponsor](https://github.com/sponsors/JetpackDuba) option and I've also added a button in the repository view as well as in my profile.

Thank you once again :smiley:
